tests/josegonzalez/Dotenv/ExpectTest.php

```diff
@@ -3,18 +3,21 @@
 namespace josegonzalez\Dotenv;
 
 use josegonzalez\Dotenv\Expect;
-use \PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase;
 
-class ExpectTest extends PHPUnit_Framework_TestCase
+class ExpectTest extends TestCase
 {
+    protected $env = [];
+    
+    protected $server = [];
 
-    public function setUp()
+    public function setUp(): void
```

Comment:
This would break PHP5 support I assume? And maybe 7?

Reply:
In fact, my changed would break all php 5.x support. The changes I'm making would end up being a new major version. only supporting 7.x, 8.x.